src/styles/main.scss

```
@@ -1,3 +1,8 @@
-body {
-  margin: 0;
-}
+@import "./utils/_variables";
+@import "./utils/_mixin";
+@import "./blocks/header.scss";
+@import "./blocks/reset.scss";
+@import "./utils/_normalize.scss";
+@import "./blocks/stars.scss";
+@import "./blocks/card.scss";
+@import "./blocks/catalog.scss";
```

```suggestion
@import "./utils/_variables";
@import "./utils/_mixin";
@import "./utils/reset.scss";
@import "./utils/_normalize.scss";

@import "./blocks/header.scss";
@import "./blocks/reset.scss";
@import "./blocks/stars.scss";
@import "./blocks/card.scss";
@import "./blocks/catalog.scss";
```
**reset.scss** has to be before **normalize.scss**